source/operatingsystems/windows/ftp/filezilla_client.md

```diff
@@ -0,0 +1,75 @@
+# Connecting to an FTP Server using FileZilla Client
+
+This guide will assist you in connecting to your FTP server using the FileZilla Client.
```

Comment:
Add a note on usage and reference to guide on installing server and configuring users to connect as?